README.md

```diff
@@ -1,23 +1,17 @@
-Beehive
-=======
+# Beehive
 
-Beehive is an event and agent system, which allows you to create your own
-agents that perform automated tasks triggered by events and filters. It is
-modular, flexible and really easy to extend for anyone. It has modules
-(we call them *Hives*), so it can interface with, talk to, or retrieve
-information from Twitter, Tumblr, Email, IRC, Jabber, RSS, Jenkins, Hue - to
-name just a few. Check out the full list of [available Hives](https://github.com/muesli/beehive/wiki/Available-Hives)
-in our Wiki.
+Beehive is an event and agent system, which allows you to create your own agents that perform automated tasks triggered by events and filters. It is modular, flexible and really easy to extend for anyone. It has modules (we call them _Hives_), so it can interface with, talk to, or retrieve information from Twitter, Tumblr, Email, IRC, Jabber, RSS, Jenkins, Hue - to name just a few. Check out the full list of [available Hives](https://github.com/muesli/beehive/wiki/Available-Hives) in our Wiki.
```

Comment:
This should be broken into lines no longer than 80 chars. In vim, highlight the line, and then run `gq` which should auto-wrap things.

Reply:
Cool. Project preferences. Will update.